src/server/replica.h

```diff
@@ -122,6 +142,8 @@ class Replica {
   // Send command, update last_io_time, return error.
   std::error_code SendCommand(std::string_view command, facade::ReqSerializer* serializer);
 
+  void ExecuteEntry(JournalExecutor* executor, journal::ParsedEntry&& entry);
+
```

Comment:
This belongs to the flow section (below StableSyncDflyFb)

Having a correctly structured header makes the code a lot easier to understand

Reply:
I did all the other changes in header apart from this one. The flow section is public and this function is private